README.md

```diff
@@ -1,132 +1,174 @@
-# neptune-r
+<div align="center">
+  <img src="https://neptune.ai/wp-content/uploads/neptune-logo-less-margin-e1611939742683.png" width="400" /><br><br>
```

Comment:
This could be translated into a Markdown image instead of HTML. I'm afraid that CRAN is not able to handle this.